docs/userguide/editing.md

```diff
@@ -27,73 +27,153 @@ rather than Google Sites.
+Markdown files are customized to work with the Jekyll static site generator,
+which benefits from [GitHub's built-in support](https://docs.github.com/en/github/working-with-github-pages/setting-up-a-github-pages-site-with-jekyll)
+for Jekyll sites.
 
-Editors are also usually ICU committers. Edit rights are granted by other Site
-owners and collaborators.
+## How to Contribute
 
-The change from editing of Open Office Writer documents and generating HTML and
-PDF to editing a Google Site simplifies the User Guide maintenance and
-encourages us to keep it more up to date than before, at the cost of not being
-able to easily generate a single PDF document with the entire contents.
+Because ICU User Guide source files are now contained within the main
+repository, changes can now be made through the same process for contributing 
+code patches. See [Contributions to the ICU Library](constributions.md) for the
+current process of filing an issue and submitting a pull request.
 
-## Document Structure
+In the case of a single file edit (ex: typo correction), a convenient way to
+create the pull request is to follow the "Edit this page on Github" link at
+the bottom of the page and [use GitHub to edit the file](https://docs.github.com/en/github/managing-files-in-a-repository/editing-files-in-your-repository)
+and create the new pull request.
 
-Major chapters have Introduction pages, and further sections in a chapter are
-subpages of that main chapter page. The navigation bar is a manually edited
-sidebar accessible (if you are logged in and have edit rights) from Site
-settings/Change appearance.
-
-Page URLs should use lowercase letters and no hyphens.
+After the source branch for a potential pull request is made, the changes can 
+be previewed using the fork repository that contains the source branch.  The
+fork repository must be [configured to serve GitHub Pages](https://docs.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#choosing-a-publishing-source)
+from the branch and the Markdown pages root directory (`/docs`).  Note: there is a
+latency of several minutes before GitHub Pages renders new changes, and 
+[errors](https://docs.github.com/en/github/working-with-github-pages/about-jekyll-build-errors-for-github-pages-sites)
+prevent the changes from being rendered.
 
-See the sitemap linked from the bottom of the navigation bar.
+## Jekyll Theme Configuration
 
-Most pages have an automatic Table of contents. On a new page, after entering
-some contents, return to the very top of the page contents, select Insert/Table
-of contents, save, then change it to Right-aligned and turn on Wrap.
+Jekyll site-wide configurations are stored
+in the `_config.yml` file. The current theme being used is
+[Just the Docs](https://pmarsceill.github.io/just-the-docs/).
 
-## Common Styles
+Due to incompatibilities between the GitHub's built-in implementation of the
```

Comment:
```suggestion
Due to incompatibilities between this theme and GitHub’s built-in implementation of the
```

Reply:
Done.  (Also, I can't tell for sure, but I think ASCII apostrophes and quotation marks in the content text are already being automatically converted in the desired curvy kind somehow.)